multinet/types.py

```diff
@@ -16,9 +16,19 @@
     },
 )
 
-Workspace = TypedDict(
-    "Workspace", {"name": str, "internal": str, "permissions": WorkspacePermissions}
-)
+
+class Workspace(TypedDict):
+    """Workspace metadata."""
+
+    name: str
+    internal: str
+    permissions: WorkspacePermissions
+
+
+class WorkspaceDocument(Workspace):
+    """Workspace metadata as it appears in ArangoDB."""
+
+    _id: str
```

Comment:
There's also the `_key` and `_rev` fields on an arango document:
```suggestion
    _id: str
    _key: str
    _rev: str
```